src/main/java/tetris/View.java

```diff
@@ -26,14 +29,18 @@
 
     /**
      * start inits the application and displays a loading screen
-     * 
+     *
      * @param primaryStage
      *            is the root object containing the application.
      */
     @Override
     public void start(Stage primaryStage) {
         settings = new Settings("src/main/resources/settings.xml");
-        controller = new Controller(this, settings);
+        TetrominoFactory factory = new TetrominoFactory();
+        SoundManager soundManager = new SoundManager(2);
+        Score score = new Score();
+        ScoreBoard scoreBoard = new ScoreBoard("src/main/resources/highscores.xml");
+        controller = new Controller(this, settings, factory, soundManager, score, scoreBoard);
```

Comment:
I would personally prefer to create the new instances of all these objects in the constructor call (which is admittedly also gnarly). Now you have all these private variables that aren't used.